.gitignore

```diff
@@ -1,4 +1,5 @@
 .tox/
 dist/
+.idea/
```

Comment:
nitpick: I prefer this to not be in each repo but in your own global gitignore

Reply:
Sounds good. I will remove it.